stack.yml

```diff
@@ -3,7 +3,13 @@ version: '3.9'
 services:
   ocurrentorg-watcher:
     image: ocurrent/watcher:latest
-    command: --github-account-allowlist="maiste" --github-app-id="232762"  --github-private-key-file="/run/secrets/ocurrentorg-key" --github-webhook-secret-file="/run/secrets/ocurrentorg-secret" --branch=live-engine -v
+    command: >
+      --github-account-allowlist="maiste"
+      --github-app-id="232762"
+      --github-private-key-file="/run/secrets/ocurrentorg-key"
+      --github-webhook-secret-file="/run/secrets/ocurrentorg-secret"
+      --branch=live-engine -v
+      --slack="/run/secret/ocurrentorg-slack"
```

Comment:
You need to map this in the `secrets` stanza below and create the secret in the docker context this pipeline is running in. See https://github.com/tarides/infrastructure/pull/188